client/src/interface/S3Api.ts

```diff
@@ -0,0 +1,5 @@
+export interface S3UploadAPIData {
+	error?: string;
```

Comment:
please fix indentation

Reply:
fixed indentation, define separate interfaces to  S3UploadAPIData and  UploadAPIDataSuccess.